### PR TITLE
2. Feature/workflow cleanup

### PR DESCRIPTION
**Pull request by arbennett**

### CODE REVIEW

File: bmorph/core/models.py [hunk 1]
```diff
@@ -267,7 +291,7 @@ def create_lstm(train_shape: tuple, depth: int=1,
     from tensorflow.keras.models import Sequential
     from tensorflow.keras.layers import Dense
     from tensorflow.keras.layers import LSTM
-    from keras.constraints import NonNeg
+    from tensorflow.keras.constraints import NonNeg
```
Comment (YifanCheng) [inline bmorph/core/models.py:294]: Is this a new syntax in tensorflow 2?

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    - scikit-learn
+    - tqdm
+    - joblib
+    - networkx
+    - graphviz
+    - pygraphviz
+    - jupyter
+    - pip
+    - pip:
+        - tensorflow
```
Comment (YifanCheng) [inline environment.yml:24]: Do you need to specify tensorlfow>=2.0.0 here? Or just specify a version?

File: bmorph/core/models.py (hunk 1 above)
Comment (arbennett) [reply bmorph/core/models.py:294]: This is to ensure that we are using `tensorflow` explicitly. Keras also is an interface for `theano`, so we want to make sure we don't import that instead.

File: environment.yml (hunk 2 above)
Comment (arbennett) [reply environment.yml:24]: Good thought - I'll update this.

**Pull request merged**